tests/test_other.py

```diff
@@ -11928,3 +11928,10 @@ def test_multiple_g_flags(self):
   def test_no_cfi(self):
     err = self.expect_fail([EMCC, '-fsanitize=cfi', '-flto', test_file('hello_world.c')])
     self.assertContained('emcc: error: emscripten does not currently support -fsanitize=cfi', err)
+
+  @also_with_wasm_bigint
+  def test_js_library_i64_params(self):
+    # Tests the defineI64Param and receiveI64ParamAsDouble helpers that are
+    # used to recieve i64 argument in syscalls.
+    self.emcc_args += ['--js-library=' + test_file('other/js_library_i64_params.js')]
+    self.do_other_test('js_library_i64_params.c')
```

Comment:
I actually think this is one of the rare cases where it make sense in `core` - it would be good to check that all opt modes work on it, including ones that minify JS and so forth.

Reply:
Mabye?   I just added a `bigint` test mode to the core test suite though.. so a test like this doesn't make sense in that mode..  but maybe it makes sense to run this at all opt levels?   Why do you think this test needs to be tests at different opt levels over any other one here in other?

Reply:
OK, I moved this into test_core.py.